src/main/scala/org/hammerlab/guacamole/commands/GermlineStandardCaller.scala

```diff
@@ -51,8 +51,6 @@ object GermlineStandard {
 
       val readSet = Common.loadReadsFromArguments(args, sc, Read.InputFilters(mapped = true, nonDuplicate = true))
       readSet.mappedReads.persist()
-      Common.progress(
-        "Loaded %,d mapped non-duplicate reads into %,d partitions.".format(readSet.mappedReads.count, readSet.mappedReads.partitions.length))
```

Comment:
There is not that much of a speed-up in dropping , the count was around 30-40 seconds of time, so we can leave it in.